config/application.rb

```diff
@@ -58,5 +58,6 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
     config.secret_token = ENV["SECRET_TOKEN"]
     config.active_record.raise_in_transactional_callbacks = true
+    config.assets.precompile << %w(landing.css)
```

Comment:
I don't think you need to make this explicit, they are automagically precompiled on deploy.

Reply:
I did this because of the upgrade of the sprocket version demands that I specify this explicitly.  It was throwing an error that landing.css was missing ....